DOCSP-27919: stableapi connection for atlas (#79)
mongodb/docs-csharp: source/fundamentals/connection/connect.txt

Connect to Atlas
~~~~~~~~~~~~~~~~

In the following example, the driver uses a sample connection URI to
connect to a MongoDB instance on Atlas, using placeholders for your

> username and password. In this example, a setting is specified for the
> client to use the :ref:`Stable API <csharp-stable-api>` feature, though
> this step is not necessary for connecting to Atlas.

Reviewer: S:

```suggestion
username and password. In this example, the client uses
the :ref:`Stable API <csharp-stable-api>` feature, though
you can connect to Atlas without this feature.
```
